src/util.jl

```diff
@@ -92,3 +92,10 @@ TrivialView(a::AbstractArray{T,N}) where {T,N} = TrivialView{typeof(a),T,N}(a)
 # """
 @inline drop_sdims(a::StaticArrayLike) = TrivialView(a)
 @inline drop_sdims(a) = a
+
+Base.@propagate_inbounds function Base.invperm(p::StaticVector)
```

Comment:
I think you need to make this just an internal-facing static arrays utility function (not an overload of `Base.invperm` because I feel that one *should* check the permutation is valid).